ArduPlane/Log.cpp

```diff
@@ -505,7 +509,7 @@ void Plane::Log_Read(uint16_t list_entry, int16_t start_page, int16_t end_page)
 
     cliSerial->printf("%s\n", HAL_BOARD_NAME);
 
-	DataFlash.LogReadProcess(list_entry, start_page, end_page,
+   DataFlash.LogReadProcess(list_entry, start_page, end_page,
```

Comment:
there are quite a few of these extraneous whitespace changes in the PR, plus a lot of fixup commits. Some commit squashing and tidying would be nice. If you are not familiar with how to do that let me know and i can do it (it is quick and easy if you know how)